src/azul/terra.py

```diff
@@ -182,6 +182,7 @@ def get_source_id(self, source: TDRSource) -> str:
         """
         Retrieve the ID of a dataset/snapshot from the TDR service API.
         """
+        assert False
```

Comment:
What's this for?

Reply:
To prove the mock can be safely removed. It's in a `drop!` commit.

Reply:
It would have helped to point this out in a comment.